src/librbd/internal.cc

```diff
@@ -1376,29 +1456,16 @@ int validate_pool(IoCtx &io_ctx, CephContext *cct) {
           return -EBUSY;
         }
       }
-    }
+      ictx->owner_lock.put_read();
 
-    BOOST_SCOPE_EXIT_ALL(ictx, cct) {
```

Comment:
Was this meant to be dropped? It looks like this code was originally dropped in Mimic as part of 86a4b33d73

Reply:
Yes, I dropped it intentionally. It was holding `owner_lock`, which would be a problem for `disable_mirroring()`. And taking that I did't see that code in newer versions and didn't have an idea why this could be needed I decided to drop it.